source/jormungandr/jormungandr/realtime_schedule/timeo.py

```diff
@@ -217,9 +218,9 @@ def _make_url(self, route_point, count=None, from_dt=None):
         # timeo can only handle items_per_schedule if it's < 5
         count = min(count or 5, 5)# if no value defined we ask for 5 passages
 
-        # if a custom datetime is provided we give it to timeo
+        # if a custom datetime is provided we give it to timeo but we round it to improve cachability
         dt_param = '&NextStopReferenceTime={dt}'\
-            .format(dt=self._timestamp_to_date(from_dt).strftime('%Y-%m-%dT%H:%M:%S')) \
+            .format(dt=floor_datetime(self._timestamp_to_date(from_dt), self.step).strftime('%Y-%m-%dT%H:%M:%S')) \
             if from_dt else ''
```

Comment:
So we don't have to modify this as it is cached on url (and it never happens)... But shouldn't we enforce that "step" and cache-duration match?

Reply:
true, I will initialize `step` with `app.config['CACHE_CONFIGURATION'].get('TIMEOUT_TIMEO', 60)`